[Feature] Classroom, Building Controller 경로별 에러 코드 문서 연결 #SCRUM-22

chaeyoungeee opened the pull request:
## ✅ 풀\_리퀘스트 체크리스트

<!--
하나씩 확인 후 체크박스에 표시해주세요.
-->

- [x] PR 제목: [Feature/Fix/Refactor...] 작업 내용 한 줄 요약 (브랜치 이름) #이슈번호
- [x] commit message 가 적절한지 확인해주세요.
- [x] 적절한 branch 로 요청했는지 확인해주세요.
- [x] Assignees, Label 을 붙여주세요.
- [x] 주의 사항과 관련해 꼭 확인해야 할 사람이 있다면 Reviewer 로 등록해주세요.
- [x] PR이 승인된 경우 해당 브랜치는 삭제해 주세요!

<br/>

## 🔄 변경 사항

- Classroom, Building Controller 경로별 에러 코드 문서와 연결하는 작업했습니다. (Univ Controller 같은 경우 확인해보니 내부 로직에서 에러가 발생하지 않아서 작업하지 않았습니다.)

- 최대 인원 에러, 단과대 값 에러, 학과 값 에러 빠져있어서 추가했습니다.

<br/>

## 📎 변경한 이유

<br/>

<!-- 관련되어있는 Issue Number 를 작성하세요! 해당 이슈를 이곳에 적으면 pr merge 이후 해당 이슈는 자동으로 close 됩니다. -->

## 👨🏻‍💻 테스트 체크리스트

<!-- 테스트 사항이 있다면 작성해 주세요! -->

<br/>

## 📌 변경 및 주의 사항

<!--
변경사항 및 주의 사항이 있다면 적어주세요.
주의 사항과 관련해 꼭 확인해야 할 사람이 있다면 리뷰어로 등록해주세요. (다른 사람이 작성한 코드 수정 등)
코드 리뷰 시 더 꼼꼼하게 확인 받고 싶은 부분이 있다면 적어주세요.
-->

<br/>

chaeyoungeee committed:
SCRUM-22 Feat: 빠진 에러 추가

chaeyoungeee committed:
SCRUM-22 Feat: Classroom 컨트롤러 에러 코드 문서 연결

chaeyoungeee committed:
SCRUM-22 Feat: Building 컨트롤러 에러 코드 문서 연결

chaeyoungeee committed:
SCRUM-22 Feat: 빠진 오류 추가

chaeyoungeee committed:
Chore: 오타 수정

Changha-dev commented:
확인하였습니다! 
수고하셨습니다 👍

Hawardshin commented on src/main/java/com/groomiz/billage/global/config/SwaggerConfig.java:
```diff
@@ -50,7 +50,7 @@ public OpenAPI openAPI() {
 			.version("1.0")
 			.description(
-				"환영합니다! [발리지](https://example.com)는 서울과학기술대학교 강의실을 빌리기 위해서 위해 만들어진 플랫폼입니다. 이 API 문서는 빌리지의 API를 사용하는 방법을 설명합니다.\n")
+				"환영합니다! [빌리지](https://example.com)는 서울과학기술대학교 강의실을 빌리기 위해서 위해 만들어진 플랫폼입니다. 이 API 문서는 빌리지의 API를 사용하는 방법을 설명합니다.\n")
```
감사합니다 ㅋ큐ㅠ

Hawardshin reviewed (approved):
고생하셨습니다!!
ExplainError 어노테이션에 ExplainError("에러에 대한 설명")
이렇게 넣으면 추가해서 사용 가능합니다! 혹시나 더 설명을 넣고 싶으면 그렇게 쓰는 것도 좋아보입니당

chaeyoungeee committed:
SCRUM-22 Chore: 오타 수정

# 본문은 아래에 작성

# 꼬릿말은 아래에 작성 (option)
# close issue #42

# --- COMMIT END ---
#   <타입> 리스트
#   Feat        : 새로운 기능 추가
#   Fix         : 버그를 고친 경우
#   Refactor    : 리팩토링
#   Comment     : 필요한 주석 추가 및 변경
#   Style       : 스타일 (코드 형식, 세미콜론 추가: 비즈니스 로직에 변경 없음)
#   Docs        : 문서 수정 (문서 추가, 수정, 삭제, README)
#   Test        : 테스트 (테스트 코드 추가, 수정, 삭제: 비즈니스 로직에 변경 없음)
#   Rename      : 파일 혹은 폴더명을 수정하거나 옮기는 작업만 한 경우
#   Remove      : 파일을 삭제하는 작업만 수행한 경우
#   Chore       : 간단한 업무지만 분류하기 애매한 경우
# ------------------
#   제목 첫 글자를 대문자로
#   제목은 명령문으로
#   제목 끝에 특수문자(.,!?) 금지
#   제목과 본문을 한 줄 띄워 분리하기
#   본문은 "어떻게" 보다 "무엇을", "왜"를 설명한다.
#   본문에 여러줄의 메시지를 작성할 땐 "-"로 구분
# ------------------
#   <꼬리말>
#   필수가 아닌 option
#   fix					:이슈 수정중 (아직 해결되지 않은 경우)
#   resolve			: 이슈 해결했을 때 사용
#   close				: 이슈 닫기
#   ex) fix: #47 resolve: #32, #21
